modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/sort/view.jsp

```diff
@@ -31,23 +29,15 @@ page import="com.liferay.portal.search.web.internal.sort.display.context.SortTer
 <%
 SortDisplayContext sortDisplayContext = (SortDisplayContext)java.util.Objects.requireNonNull(request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT));
 
-if (sortDisplayContext.isRenderNothing()) {
-	return;
-}
-
 SortPortletInstanceConfiguration sortPortletInstanceConfiguration = sortDisplayContext.getSortPortletInstanceConfiguration();
 %>
 
 <c:choose>
 	<c:when test="<%= sortDisplayContext.isRenderNothing() %>">
-		<div class="alert alert-info text-center">
-			<liferay-ui:message key="this-widget-is-not-visible-to-users-yet" />
+		<div class="alert alert-info">
+			<strong><liferay-ui:message key="sort-portlet-instance-configuration-name" />:</strong>
 
-			<clay:link
-				href="javascript:void(0);"
-				label='<%= LanguageUtil.get(request, "complete-its-configuration-to-make-it-visible") %>'
-				onClick="<%= portletDisplay.getURLConfigurationJS() %>"
-			/>
```

Comment:
I think we should still include this link? This message is different since it's asking for a configuration to be set otherwise it wouldn't be displayed.

Reply:
I'm also curious how you got this message to appear? Mine initializes with configurations already and if I remove them all it still doesn't appear. 🤔

Reply:
Oh sure! Will add the link back in. Do you think the same label is ok? I was wondering if it makes sense to ask to complete the configuration to make it visible. 😅 

To get it to appear, I went localhost:8080/search when logged in, and also I think if you try adding it to a content page it should show that message.

Reply:
Ya, if my understanding is correct, the label makes sense since it's asking the admin to define the configuration otherwise it'll never appear.